src/platform/site-wide/sass/merger.scss

```diff
@@ -175,6 +172,23 @@
     left: initial;
   }
 
+  .sign-in-nav {
+    .va-dropdown-trigger {
+
+      span {
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        margin-bottom: -6px;
+        max-width: 102px;
+
+        @media (max-width: $xsmall-screen) {
+          max-width: 60px;
+        }
+      }
+    }
+  }
+
```

Comment:
I made this more mobile-first, but I ran into issues on iPhone 5. It seems like iPhone 5 is still within the 320 width so the user name still goes of the page when the css is within the `@media (min-width: $xsmall-screen) and (max-width: $medium-screen - 1)` media query. It looks like this.

<img width="359" alt="screen shot 2018-11-05 at 5 30 00 pm" src="https://user-images.githubusercontent.com/5377648/48045816-aa8b0d00-e146-11e8-9ec7-57282774d746.png">

I needed to add the `@media (max-width: $xsmall-screen)` to target this for iPhone 5 sizes and below. This is what it looks like with the `@media (max-width: $xsmall-screen)` media query.

<img width="358" alt="screen shot 2018-11-05 at 6 38 14 pm" src="https://user-images.githubusercontent.com/5377648/48045852-d60df780-e146-11e8-9b31-28f317ef7e1e.png">

@ncksllvn @jbalboni if you can think of another way are targeting iPhone 5 please let me know.

Reply:
I think part of the problem is more that 102 pixels is too wide of a max width for screens just above 320px. It looks fine once you get to 375px, but in between there it cuts off like your screenshot above. I would do something like this:

```scss
  max-width: 60px
  @include media(@medium-phone-screen) {
    max-width: 102px;
  }
```

And set `@medium-phone-screen` to `375px` in this file. What you have now doesn't fit our existing breakpoints (each variable is intended to be a min width, not a max width) either, and I think maintaining the mobile first structure of the media queries will keep us from having problems later.